pkg/client/openstack/compute/create_options.go

```diff
@@ -1,32 +0,0 @@
-package compute
-
-import (
-	"errors"
-
-	"github.com/gophercloud/gophercloud/openstack/compute/v2/servers"
-)
-
-type CreateOpts struct {
-	servers.CreateOpts
-}
-
-func (opts CreateOpts) ToServerCreateMap() (map[string]interface{}, error) {
-	data, err := opts.CreateOpts.ToServerCreateMap()
-	if err != nil {
-		return nil, err
-	}
-	if _, ok := data["server"]; !ok {
-		return nil, errors.New("Expected field `server` not found")
-	}
-
-	serverData, ok := data["server"].(map[string]interface{})
-	if !ok {
-		return nil, errors.New("Field `server` not of expected type")
-	}
-	securityGroups := make([]map[string]interface{}, len(opts.SecurityGroups))
-	for i, groupID := range opts.SecurityGroups {
-		securityGroups[i] = map[string]interface{}{"id": groupID}
```

Comment:
Is this a "vendored" bugfix for Gophercloud?

Reply:
This was my contrived way overriding gophercloud default behaviour expecting security group names (which turns out to me the only supported way). I just removed my override to change that into an `id` field.

Reply:
So not its not a bugfix its just crap I wrote that we remove again.